test/unit/wiki_extensions_child_pages_count_test.rb

```diff
@@ -0,0 +1,28 @@
+require File.dirname(__FILE__) + '/../test_helper'
+
+class WikiExtensionsChildPagesCountTest < Redmine::HelperTest
+  include ApplicationHelper
+  include ERB::Util
+
+  fixtures :projects, :wikis, :wiki_pages, :wiki_contents, :wiki_content_versions, :issues
+
+  def test_child_pages_count_in_wiki_content
+    wiki_page = WikiPage.find(2)
+    assert_equal '<p>3</p>', textilizable('{{child_pages_count}}', object: wiki_page.content)
+  end
+
+  def test_child_pages_count_in_wiki_content_version
+    wiki_page = WikiPage.find(2)
+    assert_equal '<p>3</p>', textilizable('{{child_pages_count}}', object: wiki_page.content.versions.first)
+  end
+
+  def test_child_pages_count_in_issue_page
+    macro_error_message = "<p>" +
+                            "<div class=\"flash error\">" +
+                              "Error executing the <strong>child_pages_count</strong> macro " +
+                              "(This {{child_pages_count}} macro can be called from Wiki pages only)" +
+                            "</div>" +
+                          "</p>"
```

Comment:
ヒアドキュメントにした方がスッキリするかも。

```suggestion
    macro_error_message = <<-MESSAGE.gsub("\n", "")
<p>
<div class="flash error">
Error executing the <strong>child_pages_count</strong> macro 
(This {{child_pages_count}} macro can be called from Wiki pages only)
</div>
</p>
    MESSAGE
```
```

Reply:
fix: [5afc265](https://github.com/otegami/redmine_wiki_extensions/pull/1/commits/5afc265b7312db5230c30449f52be14f8be4b7cf): なるほどです。ヒアドキュメントにした場合に改行を置換すれば良かったのですね。パッと思いつかなかったですmm

下記の文章の前に半角スペースをいれているのは、前文の文末にいれるとdiffを見たときに赤くハイライトされてしまうのが気になるので入れています。（良い回避策があったらお聞きしたいですmm）
```ruby
 (This {{child_pages_count}} macro can be called from Wiki pages only)
^ ココです
```

Reply:
diffを見たときに赤くハイライトされるというのがどういう状況かわからないですが（なにでdiffを見ているの？エディター？エディターがtrailing spaceと認識して赤くハイライトしている？）、ここに入れても結果は変わらないのでここに入れてもいいと思います。

Reply:
ターミナルで git diff で見た場合に、おっしゃる通り、trailing spaceと認識して赤くハイライトしている状態ですmm
![Screenshot from 2024-03-12 15-23-59](https://github.com/otegami/redmine_wiki_extensions/assets/45173523/446568c0-a667-498d-abd1-ec22c0de9287)

Reply:
いや、これはそうじゃないですね。

エディターがヒアドキュメントの中のtrailing spaceも自動削除してしまっていてdiffが出ているんだと思います。エディターがもっと賢くなればいいんですけど、それは難しいんだと思うので先頭にスペースでいいと思います。